version/1/4/tl/code_of_conduct.md

```diff
@@ -0,0 +1,78 @@
+# Kodigo ng Kasunduan sa Pag-Aasal ng Contributor
+
+## Ating Panata
+
+Upang mapangalagaan ang isang bukas at mapang-anyayang komunidad, nangangako tayo
+bilang mga contributor at tagapangalaga na ang paglahok sa ating proyekto at
```

Comment:
Close enough but I don't feel it really captures the essence of the word `contributor`.

I was actually thinking of `tagapag-ambag` instead because to contribute is _mag-ambag_. But it seems like overkill. So I left the word as is.

What do you think? Would it be wise to do a word for word translation? Or should we be careful not to lose the essence of the text?